src/test/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBBySharedTableTest.java

```diff
@@ -78,84 +80,88 @@
     private static final String indexField = random("indexField");
     private static final String indexRangeField = random("indexRangeField");
 
+    private static String random(String fieldName) {
+        return fieldName + (randomFieldNames ? randomUUID().toString().replace("-", "") : "");
+    }
+
     @Test
     void sharedTableCustomDynamic() {
         CreateTableRequestFactory createTableRequestFactory = virtualTableDescription -> {
             String tableName = virtualTableDescription.getTableName();
             if (tableName.endsWith("3")) {
                 return new CreateTableRequest()
-                        .withTableName(tableName)
```

Comment:
this diff chunk here is quite tough to read...

Reply:
10 more characters in the line would have prevented this.  I can live with it though.

Reply:
oh I can live with it and I bet it's fine—I more meant to say that I've yet to actually make sure nothing was mangled by Intellij here